contextmenu/dllmain.cpp

```diff
@@ -0,0 +1,178 @@
+// dllmain.cpp : Defines the entry point for the DLL application.
+#include "pch.h"
+
+using namespace Microsoft::WRL;
+using namespace winrt::Windows::Management::Deployment;
+
+BOOL APIENTRY DllMain(HMODULE hModule,
+	DWORD  ul_reason_for_call,
+	LPVOID lpReserved
+)
+{
+	switch (ul_reason_for_call)
+	{
+	case DLL_PROCESS_ATTACH:
+	case DLL_THREAD_ATTACH:
+	case DLL_THREAD_DETACH:
+	case DLL_PROCESS_DETACH:
+		break;
+	}
+	return TRUE;
+}
+
+class ExplorerCommandBase : public RuntimeClass<RuntimeClassFlags<ClassicCom>, IExplorerCommand, IObjectWithSite> {
```

Comment:
Not sure if `IObjectWithSite` is actually needed here. In the sample project it is added to show message boxes as Explorer. I've tested removing `IObjectWithSite` on another project and it worked.

Reply:
I'll try and see about that, thanks

Reply:
I have removed it now